<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorMainWindow</class>
 <widget class="QMainWindow" name="DoctorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DrStrange v.1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

#frame{
background:rgba(112, 127, 158,0.6);
color: white;
}

#frame_7{
border: 2px solid gray;
border-radius: 10px;
}

#frame_8{
border: 2px solid gray;
border-radius: 10px;
}

#frame_9{
border: 2px solid gray;
border-radius: 10px;
}

#frame_2{
border: 2px solid gray;
border-radius: 10px;
}

</string>
  </property>
  <widget class="QWidget" name="centralDoctorMainwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
       <item>
        <widget class="QFrame" name="drMainTopFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,2,8">
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="1">
              <widget class="QLabel" name="userIDLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="userName">
               <property name="text">
                <string>Doctor ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="DoctorWindouserNameLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>User ID</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Hospital ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="hospitalIDLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="drNameLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_9">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="logoutButton">
               <property name="text">
                <string>Logout</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Appointments</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>583</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="apointmentsTab">
             <attribute name="title">
              <string>Appointments</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QTableView" name="appointmentsTableView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="SearchPatientTab">
             <attribute name="title">
              <string>SearchPatient</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QFrame" name="frame_5">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QFrame" name="frame_3">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="2" column="1">
                     <widget class="QLineEdit" name="SSNLine"/>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="FirstNameLabel">
                      <property name="text">
                       <string>First Name</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="DOBLabel">
                      <property name="text">
                       <string>Date of Birth</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLineEdit" name="DOBLine"/>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLineEdit" name="LastNameLine"/>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="LastNameLabel">
                      <property name="text">
                       <string>Last Name</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="SSNLabel">
                      <property name="text">
                       <string>SSN Last 4 #</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="FirstNameLine"/>
                    </item>
                    <item row="4" column="0">
                     <widget class="QPushButton" name="SearchButton">
                      <property name="text">
                       <string>Search</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLabel" name="searchResultLabel">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_4">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QTableView" name="PatientQueryTable"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="AddPatientTab">
             <attribute name="title">
              <string>AddPatient</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="1" column="0" colspan="2">
                  <widget class="QLabel" name="LastNameLabel2">
                   <property name="text">
                    <string>Last Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QLabel" name="LastNameLabel2_2">
                   <property name="text">
                    <string>Middle Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="9" column="2">
                  <widget class="QTextEdit" name="textEdit_2"/>
                 </item>
                 <item row="3" column="4">
                  <widget class="QLineEdit" name="addPatientWorkPhone">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="3">
                  <widget class="QLabel" name="WorkPhone">
                   <property name="text">
                    <string>Work Phone</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="0">
                  <widget class="QLabel" name="ZipCode">
                   <property name="text">
                    <string>Zip Code</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <widget class="QLabel" name="DOB2">
                   <property name="text">
                    <string>DOB</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="4">
                  <widget class="QLineEdit" name="addPatientDOB">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <widget class="QLineEdit" name="addPatientName">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="City">
                   <property name="text">
                    <string>City</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="Address1">
                   <property name="text">
                    <string>Address 1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="2">
                  <widget class="QLineEdit" name="addPatientCity">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="8" column="0">
                  <widget class="QLabel" name="FirstNameLabel2_2">
                   <property name="text">
                    <string>Allergy History</string>
                   </property>
                  </widget>
                 </item>
                 <item row="10" column="0">
                  <widget class="QLabel" name="FirstNameLabel2_4">
                   <property name="text">
                    <string>Diagonistics</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="MobilePhone">
                   <property name="text">
                    <string>Mobile Phone</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="3">
                  <widget class="QLabel" name="EmailAddress">
                   <property name="text">
                    <string>Email Address</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="3">
                  <widget class="QLabel" name="State">
                   <property name="text">
                    <string>State</string>
                   </property>
                  </widget>
                 </item>
                 <item row="10" column="2">
                  <widget class="QTextEdit" name="textEdit_3"/>
                 </item>
                 <item row="5" column="3">
                  <widget class="QLabel" name="Address2">
                   <property name="text">
                    <string>Address 2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QLineEdit" name="addPatientHomePhone">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="2">
                  <widget class="QLineEdit" name="addPatientZip">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="8" column="2">
                  <widget class="QTextEdit" name="textEdit"/>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLineEdit" name="addPatientLastName">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="9" column="0">
                  <widget class="QLabel" name="FirstNameLabel2_3">
                   <property name="text">
                    <string>Lab History</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QLineEdit" name="addPatientMiddleName">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="HomePhone">
                   <property name="text">
                    <string>Home Phone</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="2">
                  <widget class="QLineEdit" name="addPatientAddressLine1">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="4">
                  <widget class="QLineEdit" name="addPatientAddressLine2">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="4">
                  <widget class="QLineEdit" name="addPatientEmail">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QLineEdit" name="addPatientSSN">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="FirstNameLabel2">
                   <property name="text">
                    <string>First Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="4">
                  <widget class="QLineEdit" name="addPatientState">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="SSN2">
                   <property name="text">
                    <string>SSN#</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QLineEdit" name="addPatientMobile">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="11" column="0">
                  <widget class="QPushButton" name="addPatientSaveButton">
                   <property name="text">
                    <string>Save</string>
                   </property>
                  </widget>
                 </item>
                 <item row="11" column="2">
                  <widget class="QLabel" name="addPatientResultLabel">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDoctor">
    <property name="title">
     <string>Doctor</string>
    </property>
   </widget>
   <addaction name="menuDoctor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>FirstNameLine</tabstop>
  <tabstop>LastNameLine</tabstop>
  <tabstop>SSNLine</tabstop>
  <tabstop>DOBLine</tabstop>
  <tabstop>SearchButton</tabstop>
  <tabstop>addPatientName</tabstop>
  <tabstop>addPatientMiddleName</tabstop>
  <tabstop>addPatientLastName</tabstop>
  <tabstop>addPatientDOB</tabstop>
  <tabstop>addPatientSSN</tabstop>
  <tabstop>addPatientEmail</tabstop>
  <tabstop>addPatientMobile</tabstop>
  <tabstop>addPatientWorkPhone</tabstop>
  <tabstop>addPatientHomePhone</tabstop>
  <tabstop>addPatientAddressLine1</tabstop>
  <tabstop>addPatientAddressLine2</tabstop>
  <tabstop>addPatientCity</tabstop>
  <tabstop>addPatientState</tabstop>
  <tabstop>addPatientZip</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>textEdit_2</tabstop>
  <tabstop>textEdit_3</tabstop>
  <tabstop>addPatientSaveButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>PatientQueryTable</tabstop>
  <tabstop>logoutButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>appointmentsTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
